### PR TITLE
Release Notes for Release SDK 120

### PR DESCRIPTION
**Pull request by mikehoffms**
Rendered article section for review:
https://review.learn.microsoft.com/microsoft-edge/webview2/release-notes?branch=pr-en-us-2953&tabs=dotnetcsharp#10221055

AB#47968051

### CODE REVIEW

**Review by victorhuangwq — APPROVED** [review]
LGTM

File: microsoft-edge/webview2/release-notes.md [hunk 1]
```diff
@@ -357,7 +357,7 @@ The following APIs have been promoted to Stable and are now included in this Rel
 
 * Added support for promise cancellation of async methods in WinRT.  (Runtime and SDK)
 
-* Disabled HTTPS upgrades to WebView2 API navigations.  (Runtime-only)  ([Issue #4104](https://github.com/MicrosoftEdge/WebView2Feedback/issues/4104))
+* Disabled auto HTTPS upgrades for WebView2 API navigations.  (Runtime-only)  ([Issue #4104](https://github.com/MicrosoftEdge/WebView2Feedback/issues/4104))
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:360]:
```suggestion
* Disabled automatic HTTPS upgrades for WebView2 API navigations.  (Runtime-only)  ([Issue #4104](https://github.com/MicrosoftEdge/WebView2Feedback/issues/4104))

```

**Review by mikehoffms — COMMENTED** [review]
This PR LGTM

File: microsoft-edge/webview2/release-notes.md [hunk 2]
```diff
@@ -138,6 +138,230 @@ The following APIs have been promoted from Experimental to Stable in this Prerel
...
+
+* [ICoreWebView2ProfileDeletedEventHandler](/microsoft-edge/webview2/reference/win32/icorewebview2profiledeletedeventhandler?view=webview2-1.0.2210.55&preserve-view=true)<!-- handler is Win32-only -->
+
+---
+
+
+<!-- ------------------------------ -->
+#### Bug fixes
+
+* Added support for promise cancellation of async methods in WinRT.  (Runtime and SDK)
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/release-notes.md:358]: @victorhuangwq Could you please check with Steven? IIUC this is more specific to wv2winrt and host objects.

**Review by peiche-jessica — APPROVED** [review]

File: microsoft-edge/webview2/release-notes.md (hunk 2 above)
Comment (victorhuangwq) [reply microsoft-edge/webview2/release-notes.md:358]: https://microsoft.visualstudio.com/Edge/_git/a45cf45f-f6db-438b-baf7-da6b3de589ba/commit/55fb27b89692b31e7852c2462f88012b3427e547?tab=details

**Pull request merged**